### PR TITLE
[14.0] [FIX] purchase_advance_payment: Wrong residual amt when reconciling credit note

### PR DESCRIPTION
**Pull request by ygol**

### CODE REVIEW

**Comment by ygol** [comment]
Hi @HviorForgeFlow @MateuGForgeFlow  do you think what I did fix the problem without introducing unwanted behavior?

**Comment by MiquelRForgeFlow** [comment]
@ygol some tests are failing

**Comment by ygol** [comment]
> @ygol some tests are failing

@MiquelRForgeFlow I've fixed the test but now getting red on coverage and don't know what to do about it. do you think my solution fix the problem? TIA

**Comment by MiquelRForgeFlow** [comment]
@ygol The coverage in red means that it needs more tests for the parts of the code which previous tests don't already cover. But the coverage codecov ci is not required, so it is fine as it is now. But you have done several commits. It would be better if you squash them and also put a meaningful message in the commit following the [guidelines](https://www.odoo.com/documentation/16.0/contributing/development/git_guidelines.html#tag-and-module-name) :)

**Comment by ygol** [comment]
> @ygol The coverage in red means that it needs more tests for the parts of the code which previous tests don't already cover. But the coverage codecov ci is not required, so it is fine as it is now. But you have done several commits. It would be better if you squash them and also put a meaningful message in the commit following the [guidelines](https://www.odoo.com/documentation/16.0/contributing/development/git_guidelines.html#tag-and-module-name) :)

Thanks for the guidance @MiquelRForgeFlow
all is ok now?

**Review by MateuGForgeFlow — APPROVED** [review]

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

**Comment by HviorForgeFlow** [comment]
ping @rvalyi @clementmbr @PierrickBrun @NathanQj

**Review by HviorForgeFlow — CHANGES_REQUESTED** [review]
@ygol seems that 87bd4b64934032098ebc3e861a45d5ffc85044dc has more changes that the one specified in the commit regarding FR translation on other modules.

**Comment by ygol** [comment]
> @ygol seems that [87bd4b6](https://github.com/OCA/purchase-workflow/commit/87bd4b64934032098ebc3e861a45d5ffc85044dc) has more changes that the one specified in the commit regarding FR translation on other modules.

@HviorForgeFlow I don't understand. what should be done?

**Comment by HviorForgeFlow** [comment]
@ygol drop these translation from the commit itself:

image

**Comment by MiquelRForgeFlow** [comment]
> drop these translation from the commit itself:

Or maybe put them in a different commit.

File: purchase_advance_payment/__manifest__.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 {
     "name": "Purchase Advance Payment",
-    "version": "14.0.1.0.1",
+    "version": "14.0.1.1.0",
```
Comment (HviorForgeFlow) [inline purchase_advance_payment/__manifest__.py]: It is not required to update version manually, ocabot will do it.

File: README.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ addon | version | maintainers | summary
 [purchase_cancel_reason](purchase_cancel_reason/) | 14.0.1.0.0 |  | Purchase Cancel Reason
 [purchase_commercial_partner](purchase_commercial_partner/) | 14.0.1.0.0 |  | Add stored related field 'Commercial Supplier' on POs
 [purchase_delivery_split_date](purchase_delivery_split_date/) | 14.0.1.1.1 |  | Allows Purchase Order you confirm to generate one Incoming Shipment for each expected date indicated in the Purchase Order Lines
-[purchase_deposit](purchase_deposit/) | 14.0.1.1.1 |  | Option to create deposit from purchase order
+[purchase_deposit](purchase_deposit/) | 14.0.1.1.2 |  | Option to create deposit from purchase order
```
Comment (HviorForgeFlow) [inline README.md:37]: This change shouldn't be in the commit.

**Review by HviorForgeFlow — CHANGES_REQUESTED** [review]
Minor comments

File: README.md (hunk 2 above)
Comment (MiquelRForgeFlow) [reply README.md:37]: This will be later overwritten by the bot.

Comment (ygol) [reply README.md:37]: so all is ok for now?

**Review by HviorForgeFlow — APPROVED** [review]

**Pull request merged**